.github/renovate.json

```diff
@@ -31,13 +37,29 @@
         "pascalcase",
         "pretty-bytes",
         "pretty-ms",
+        "stdout-update",
         "tempy",
         "terminal-link"
       ],
       "matchPackagePatterns": [
         "^@redwoodjs/"
-      ],
-      "enabled": false
+      ]
+    },
+
+    {
+      "groupName": "chore",
+
+      "matchPackageNames": [
+        "all-contributors-cli",
+        "cypress",
+        "cypress-wait-until",
+        "dependency-cruiser",
+        "glob",
+        "mheap/github-action-required-labels action",
+        "nx",
+        "sort-package-json",
+        "zx"
+      ]
     }
```

Comment:
Lets renovate group updates to these dependencies. Smart grouping will save us some noise; I intend to try it out with more packages where it makes sense